src/main/java/net/minecraftforge/common/MinecraftForge.java

```diff
@@ -64,7 +64,10 @@ public static void initialize()
      *                       action of your close button, using {@link Screen#minecraft} to get the client instance.</p>
      * @see ModLoadingContext#registerExtensionPoint(Class, Supplier)
      * @see ModLoadingContext#registerConfig(ModConfig.Type, IConfigSpec)
+     *
+     * Deprecated, take a look at {@link net.minecraftforge.fml.event.lifecycle.FMLConstructModEvent}
      */
+    @Deprecated(forRemoval = true)
     public static void registerConfigScreen(Function<Screen, Screen> screenFunction) {
```

Comment:
Currently doesn't render correctly:
<img width="587" alt="image" src="https://github.com/user-attachments/assets/d074a2ea-bc19-4973-8cc6-9217d0cb9953">

```suggestion
     *
     * @deprecated Use {@link net.minecraftforge.fml.event.lifecycle.FMLConstructModEvent#registerConfigScreen(Function)} instead
     */
    @Deprecated(forRemoval = true)
    public static void registerConfigScreen(Function<Screen, Screen> screenFunction) {
```